shared/domain/src/main/java/com/ivy/domain/usecase/BalanceBuilder.kt

```diff
@@ -0,0 +1,56 @@
+package com.ivy.domain.usecase
+
+import com.ivy.data.model.primitive.AssetCode
+import com.ivy.data.model.primitive.NonZeroDouble
+import com.ivy.data.model.primitive.PositiveDouble
+
+class BalanceBuilder {
+
+    private val balance = mutableMapOf<AssetCode, NonZeroDouble>()
+
+    fun processIncomes(
```

Comment:
processAssetsIn + add some KDoc explaining what it does

Reply:
actually nvm, it's a bad name suggestion

Reply:
Ok